raiden/tests/test_accounts.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+import os
+import pytest
+
+from ethereum.utils import encode_hex
+from raiden.accounts import AccountManager
+from raiden.utils import get_project_root
+
+test_keystore = os.path.join(get_project_root(), 'tests', 'test_files')
+
+
+def test_get_accounts():
+    accmgr = AccountManager(test_keystore)
```

Comment:
it becomes a bit too verbose in my opinion, but sure I can change it if you want